Failure Modules/TestFlightFailure_SolarTracking.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using KSP;
+
+namespace TestFlight.Failure_Modules
+{
+    public class TestFlightFailure_SolarTracking : TestFlightFailureBase_Solar
+    {
+        public override void DoFailure()
+        {
+            base.DoFailure();
+            base.module.trackingSpeed = 0;
+        }
+        public override float DoRepair()
+        {
+            base.DoRepair();
+            if (base.part != null && base.part.partInfo != null)
+            {
+                Part pf = base.part.partInfo.partPrefab;
+                if (pf != null)
+                {
+                    ModuleDeployableSolarPanel pm = (ModuleDeployableSolarPanel)pf.Modules.OfType<ModuleDeployableSolarPanel>();
+                    if (pm != null)
+                    {
+                        base.module.trackingSpeed = pm.trackingSpeed;
```

Comment:
That scenario would only happen in the very specific case of

1. Part fails
2. Game is saved
3. New mod is installed changing default value
4. Game with failed part is loaded

While it certainly can happen, it is definitely in the minority.  On the other hand what if the player has another mod installed that is changing tracking speed at runtime based on something, or even what if you or someone else makes another failure mod that slows tracking speed rather than stops it.  In both cases, if you set the tracking speed back to the prefab value, that would be the wrong value.  But if you cache it, you will always know you are setting it back to what it was immediately prior to failure.

In other words caching it is going to give you a "fixed" value more often than not, and as such it is the general rule I use when modifying values at runtime in failures.